cdc/processor/manager.go

```diff
@@ -127,10 +127,6 @@ func (m *managerImpl) Tick(stdCtx context.Context, state orchestrator.ReactorSta
 		})
 		if err := p.Tick(ctx); err != nil {
 			m.closeProcessor(changefeedID)
-			if cerrors.ErrReactorFinished.Equal(errors.Cause(err)) {
-				continue
-			}
-			return state, errors.Trace(err)
```

Comment:
we should return processor error no matter what it is.

Reply:
Can I ask why?

Reply:
It is verbose, we don't need this `if` condiction indeed, since we should never return an processor error here.

Reply:
In the original logic, all errors return from `p.tick()` would be wrapped into a `ErrReactorFinished` error, as you can see in line:551, so all error return to here will be catch and the loop should continue. 
In addition, the processor's `handleErr()` has already patch a `taskPosition` to tell the owner it encounters an error,  we can just let go here.

Reply:
@3AceShowHand  please see: https://github.com/pingcap/tiflow/pull/7216/files#diff-a92d89dfa9f192c1a4325f9600f7df6fb03de4361e37d9107982d585592c632eR550